### DISCUSSION TITLE
Superset insert data to sqlite3 rather than postgresql

### DISCUSSION THREAD

**Discussion by yehaotong** [apache/superset#24463] [Q&A / Help]
I use the docker-compose-non-dev.yml to deploy a superset cluster, and I create some user and some chart on superset web, and then I can not find the data on the postgresql:superset table, I think this data is stored in its own sqlite3 database. If I want to save the data in PostgreSQL db, what configuration should I modify on the docker-compose-non-dev.yml or env？

**Comment by sebastianliebscher** [comment]
Hi @yehaotong! For questions regarding the use of Superset you can use the GitHub [Discussions](https://github.com/apache/superset/discussions) or join the [Superset Slack](http://bit.ly/join-superset-slack). There you can share more information and ask for help.

**Comment by Huy-Ngo** [comment]
Having the same issue here: I set environment variables `DATABASE_{DIALECT,HOST,USERNAME,PASSWORD}` and `SQLALCHEMY_DATABASE_URI` expecting it to use postgresql but it keeps using sqlite (and fail):

```
superset-init-1  | 2025-03-07 08:34:57,497:ERROR:flask_appbuilder.security.sqla.manager:DB Creation and initialization failed: (sqlite3.OperationalError) unable to open database file
```
